pkgs/by-name/ro/rosa/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles, stdenv, testers, rosa, go }:
+
+buildGoModule rec {
+  pname = "rosa";
+  version = "1.2.35";
+
+  src = fetchFromGitHub {
+    owner = "openshift";
+    repo = "rosa";
+    rev = "v${version}";
+    hash = "sha256-QMoUies5siHA9AlXLilz1/i6YTE8Z4HHO1BsyJnRkrc=";
+  };
+  vendorHash = null;
+
+  ldflags = [ "-s" "-w" ];
+
+  __darwinAllowLocalNetworking = true;
+
+  nativeBuildInputs = [ installShellFiles ];
+  postInstall = ''
+    installShellCompletion --cmd rosa \
+      --bash <($out/bin/rosa completion bash) \
+      --fish <($out/bin/rosa completion fish) \
+      --zsh <($out/bin/rosa completion zsh)
+  '';
+
+  preCheck = ''
+    ln -s $GOPATH/bin/rosa rosa
+  '';
```

Comment:
How does that interact with go test?

Reply:
That is something I copied from a related package 
https://github.com/NixOS/nixpkgs/blob/95002f7df788ca7b89fb16cd5241345c33839035/pkgs/applications/networking/cluster/ocm/default.nix#L22-L24
without really knowing why it was needed or even if this new package needed it. Looks like it does not. Lesson learned :)
removed in 1839dd30e1d5